README.md

```diff
@@ -1542,6 +1542,7 @@ _Before hosting one, please see [shortcomings](https://en.wikipedia.org/wiki/URL
 
 **[`^        back to top        ^`](#)**
 
+- [Blink](https://github.com/JaneJeon/blink) - easy-to-host, SSO-integrated, CDN-powered link shortener (+decoupled analytics) for teams ([Docs](https://blink.rest)) `AGPL-3.0` `Nodejs`
```

Comment:
- missing final dot after `teams`
- missing capitalization on first word `Easy`
- the main link should be https://docs.blink.rest/
- the source code link should be https://github.com/JaneJeon/blink
- `Docs` is not a valid link title

Reply:
@JaneJeon ping, can you fix the points mentioned above? thanks

Reply:
sorry, I've just been distracted by other things. Will fix!

Reply:
Okay, I've just fixed them!